### ISSUE TITLE
Bugs, many bugs

### ISSUE THREAD

**Issue by canedoly** [Fedoraware/Fedoraware#622]
- Bullet tracers on projectile's sometimes renders twice or more (this bug appears much more often with dt and moving enemies)
- Bullet tracers don't have glow (a major issue)
- Fake angles kinda teleport back like 8 ticks when moving if you have no fakelag
- In rijin DT indicator the text's outline is overlapping with the border, which is more visible with colors like blue
- Recharging while charging as demoman causes you to stop instantly
- Unlocking achievements unlocks only like 5 (it should atleast unlock all milestone achievements)
- Rarely bullet tracers origin is like far far away
- Very rarely the sticky aimbot will fail to shoot at the target (it will just shoot where your crosshair is)
- Very rarely bullet tracers are inaccurate with projectiles

I've came across all these bugs in 2 hours of playing
I will try to give recordings to all the glitches i've mentioned, and the config later today

Suggestion
Add ability to snap align your dt bar and other stuff so you can have it perfectly in the middle or in other postitions

**Comment by lnx00** [comment]
> Unlocking achievements unlocks only like 5

You have to run it in the main menu while not being connected to a server. It did work for me this way.

**Comment by johnathon-walnut** [comment]
Bullet tracers on projectiles doubling - I can fix
Bullet tracers not having glow - HUGE ISSUE, MUST FIX ASAP
Fake angles kinda teleport back like 8 ticks when moving if you have no fakelag - Dunno, haven't really noticed but I know that the fake angle chams are WACKY
The Rijin DT indicator is dependent on your indicator font
Recharging while charging as demoman is because for some reason Baan thinks its cool to stop movement when recharging
Unlocking achievements - unlock in the main menu
Rarely bullet tracers origin is like far far away - honestly, haven't seen this one ever
Sticky aimbot - /shrug
Bullet tracers - /shrug

Suggeston - I did suggest this when Baan was redoing my thing in imgui to add a context menu to put something in the center, might look into it

**Comment by johnathon-walnut** [comment]
fixed the major issue here

**Comment by xxmyn** [comment]
@canedoly what do you mean by fake angle chams teleporting by 8 ticks?

**Comment by canedoly** [comment]
> @canedoly what do you mean by fake angle chams teleporting by 8 ticks?

It's hard to explain this in words but the fake angle chams without fakelag sometimes render like 6-14 ticks behind our original position, its kinda weird I'll provide the screenshot later if I won't forget like last time

So I just tested and there are still bugs
- Bullet tracers still sometimes renders twice
- Bullet tracers sometimes don't even render
- Bullet tracers sometimes are inaccurate asf (most often with demoman)
- Bullet tracers origin sometimes is like not even close
- Recharging still breaks while charging as demoman

**Comment by Capybaron** [comment]
> > @canedoly what do you mean by fake angle chams teleporting by 8 ticks?
> 
> It's hard to explain this in words but the fake angle chams without fakelag sometimes render like 6-14 ticks behind our original position, its kinda weird I'll provide the screenshot later if I won't forget like last time
> 
> So I just tested and there are still bugs
> 
> * Bullet tracers still sometimes renders twice
> * Bullet tracers sometimes don't even render
> * Bullet tracers sometimes are inaccurate asf (most often with demoman)
> * Bullet tracers origin sometimes is like not even close
> * Recharging still breaks while charging as demoman

who

**Comment by canedoly** [comment]
> > > @canedoly what do you mean by fake angle chams teleporting by 8 ticks?
> > 
> > 
> > It's hard to explain this in words but the fake angle chams without fakelag sometimes render like 6-14 ticks behind our original position, its kinda weird I'll provide the screenshot later if I won't forget like last time
> > So I just tested and there are still bugs
> > 
> > * Bullet tracers still sometimes renders twice
> > * Bullet tracers sometimes don't even render
> > * Bullet tracers sometimes are inaccurate asf (most often with demoman)
> > * Bullet tracers origin sometimes is like not even close
> > * Recharging still breaks while charging as demoman
> 
> who

me

**Comment by Capybaron** [comment]
cares

**Comment by johnathon-walnut** [comment]
> > @canedoly what do you mean by fake angle chams teleporting by 8 ticks?
> 
> It's hard to explain this in words but the fake angle chams without fakelag sometimes render like 6-14 ticks behind our original position, its kinda weird I'll provide the screenshot later if I won't forget like last time
> 
> So I just tested and there are still bugs
> 
>     * Bullet tracers still sometimes renders twice
> 
>     * Bullet tracers sometimes don't even render
> 
>     * Bullet tracers sometimes are inaccurate asf (most often with demoman)
> 
>     * Bullet tracers origin sometimes is like not even close
> 
>     * Recharging still breaks while charging as demoman

The bullet tracers are perfect, are you talking about the ones from projectile weapons? (They are done slightly different)

**Comment by canedoly** [comment]
> > > @canedoly what do you mean by fake angle chams teleporting by 8 ticks?
> > 
> > 
> > It's hard to explain this in words but the fake angle chams without fakelag sometimes render like 6-14 ticks behind our original position, its kinda weird I'll provide the screenshot later if I won't forget like last time
> > So I just tested and there are still bugs
> > ```
> > * Bullet tracers still sometimes renders twice
> > 
> > * Bullet tracers sometimes don't even render
> > 
> > * Bullet tracers sometimes are inaccurate asf (most often with demoman)
> > 
> > * Bullet tracers origin sometimes is like not even close
> > 
> > * Recharging still breaks while charging as demoman
> > ```
> 
> The bullet tracers are perfect, are you talking about the ones from projectile weapons? (They are done slightly different)

Yeah, the hitscan works perfectly fine meanwhile projectile has some weird bugs

**Comment by canedoly** [comment]
So i've found another bug
- While using antiaim, and holding melee and trying to attack without aimbot, melee will just attack where the anti aim is looking.

Suggestion
- Team based chams

Also randomly my backtrack chams went black and glow stopped working, I've tried using f_dumpmaterials but the list was so big it didn't fit in the console

**Comment by AscendedSion** [comment]
this is not a problem with cuntyhack! (probably  is)

**Comment by xxmyn** [comment]
> So i've found another bug
> 
> - While using antiaim, and holding melee and trying to attack without aimbot, melee will just attack where the anti aim is looking.
> 
> 
> 
> Suggestion
> 
> - Team based chams
> 
> - And bring back [this](https://github.com/Fedoraware/Fedoraware/commit/6baa917bf73821aca745ff318a2650e60591c8a7), it looked cool 
> 
> 
> 
> Also randomly my backtrack chams went black and glow stopped working, I've tried using f_dumpmaterials but the list was so big it didn't fit in the console


that debug fakelag "feature" was an attempt of mine to fuck with animations, i removed it because it broke the anim cycle for any animation that wasnt your primary animation (layers like attacking would take like 30 seconds to complete) and even after iterating through layers i couldnt fix it

**Comment by johnathon-walnut** [comment]
We should probably make the dumpallmaterials write to a file lol

**Comment by canedoly** [comment]
I've found some more bugs, and i've edited my recent comment
